docs/modeling/new.rst

```diff
@@ -327,6 +327,30 @@ necessary::
                                              ineqcons=self.ineqcons)
         return model_copy
 
+Defining a Plugin Fitter
+------------------------
+
+The is now a plugin mechanism which allows fitters defined outside of
```

Comment:
In general one should avoid time references like "now" in docs because in 2 or 10 years that doc line will still probably be there.

So say "Astropy.modeling includes a plugin .."